tests/test_C1_verification.py

```diff
@@ -33,6 +38,45 @@ def test_lookup_file_non_existing_file_raises_error():
         C1.lookup_file(file_path=file_name, name="test")
 
 
+def test_check_if_energy_vector_is_defined_in_DEFAULT_WEIGHTS_ENERGY_CARRIERS_pass():
+    # Function only needs to pass
+    C1.check_if_energy_vector_is_defined_in_DEFAULT_WEIGHTS_ENERGY_CARRIERS(
+        "Electricity", "asset_group", "asset"
+    )
+    assert (
+        1 == 1
+    ), f"The energy carrier `Electricity` is not recognized to be defined in `DEFAULT_WEIGHTS_ENERGY_CARRIERS`."
+
+
+def test_check_if_energy_vector_is_defined_in_DEFAULT_WEIGHTS_ENERGY_CARRIERS_fail():
+    with pytest.raises(UnknownEnergyVectorError):
+        C1.check_if_energy_vector_is_defined_in_DEFAULT_WEIGHTS_ENERGY_CARRIERS(
+            "Bio-Diesel", "asset_group", "asset"
+        ), f"The energy carrier `Bio-Diesel` is recognized in the `DEFAULT_WEIGHTS_ENERGY_CARRIERS`, eventhough it should not be defined."
+
+
+def test_check_if_energy_vector_of_all_assets_is_valid_passes():
+    dict_test = {
+        PROJECT_DATA: {LES_ENERGY_VECTOR_S: {"Electricity"}},
+        ENERGY_PRODUCTION: {ENERGY_VECTOR: "Electricity"},
+    }
+    C1.check_if_energy_vector_of_all_assets_is_valid(dict_test)
+    assert (
+        1 == 1
```

Comment:
need to add a TODO statement here, or remove the useless test

Reply:
But this checks that the function does not terminate / the test passes - isn't that how you test that?

Reply:
I think you can remove the assert and the test is still going to be ran

Reply:
True... but  I want to see the assertion debug message if it fails, so I will leave it in.